src/hooks/useProjects.tsx

```diff
@@ -68,5 +69,16 @@ export const useProjects = () => {
     });
   };
 
-  return { tableColumns, tableData };
+  const metaData = (data: any) => {
+    return {
```

Comment:
@nathankanyesigye Can we also include the `total projects` part of this overview metadata to be uniform with metadata of similar sections i.e. Users, Databases, and Apps listings for the admin.

Check the response for the `total_projects` key to get its value.